setup_root.md

```diff
@@ -1,65 +1,211 @@
+Also, this will create migration files. By default, every migration file has two functions, namely up and down. The up function creates a table, the down function deletes the existing table. Furthermore it is possible to customize the migration functions. Please check [this](zendro_cli.md) to learn more about migrations.
 
-### Step 4: Start up your Zendro instance
+***WHEN NOT TO USE -m AND CONNECT TO OWN DB IS MISSING*** 
 
+### Step 6: Start up Zendro 
+
+<!----><a name="start"></a>
+#### Using docker
 The recommend way to [run your Zendro instance is via docker](https://zendro-dev.github.io/zendro_cli.html#dockerize-zendro-app-with-example-docker-files). This ensures that regardless of your local infrastructure Zendro will behave the same.
 ```
-zendro dockerize -u
+$ zendro dockerize -u 
+```
+
+Moreover, if you would like to use production mode, please execute:
+```
+$ zendro dockerize -u -p
+```
+
+This command will create docker containers for each Zendro component:
+* [Keycloak](https://github.com/Zendro-dev/Zendro-dev.github.io/blob/documentation-vb/oauth.md): manage users and roles
+* [Single Page App (SPA)](https://github.com/Zendro-dev/single-page-app): graphical interface to send CRUD requests to a Zendro GraphQL endpoint
+* [API](https://github.com/Zendro-dev/graphql-server): CRUD API that can be accessed through a GraphQL query language
+* [API with authenthication](https://github.com/Zendro-dev/graphiql-auth): An implementation of the GraphQL IDE with Zendro login
+
+You can check docker containers by:
+```
+$ docker ps
 ```
-It is also possible to [run the zendro services locally](https://zendro-dev.github.io/zendro_cli.html#start-zendro-service), however you might run into unexpected incompatibilities depending on your local system.
+
+You can check docker logs by:
 ```
-zendro start
+$ docker logs -f <container name>
 ```
 
-### Step 5 (optional): Customize your project
+> ***Please wait until logs indicate the app is running on XXXX port to access Zendro services.***
+
+In default config, the running containers will be on ports:
+
+* Keycloak: http://10.5.0.11:8081
+   * The default keycloak username is *admin* and the password is *admin*.
+
+  ![Keycloak example](figures/kc1.png)
+  ![Keycloak example](figures/kc2.png)
+
+* SPA: http://localhost:8080
+    * The default zendro username is *zendro-admin* and the password is *admin*.
+
+  ![spa example](figures/login.png)
+  ![spa example](figures/spa.png)
+
+* GraphQL API: http://localhost:3000/graphql
+
+  ![api example](figures/graphql.png )
+
+* GraphQL API with authenthication: http://localhost:7000
+    * The default zendro username is *zendro-admin* and the password is *admin*.
+
+  ![api example](figures/login.png)
+  ![api example](figures/graphiql.png)
+
+
+If you wish to modify the default ports, adjust next files:
+> [Stop Zendro Instance](#stop), modify files and [start Zendro Instance](#start) again.
```

Comment:
how to stop zendro? cannot switch to the anchor

Reply:
Can you switch to start zendro anchor?